crates/matrix-sdk-crypto/src/secret_storage.rs

```diff
@@ -754,6 +754,24 @@ mod test {
         );
     }
 
+    /// The `iv` and `mac` properties within the `m.secret_storage.key.*`
+    /// content are optional, and the spec says we must assume the
+    /// passphrase is correct in that case.
+    #[test]
+    fn invalid_key_with_no_mac() {
```

Comment:
Can we make this into a sentence? Maybe `invalid_key_with_no_mac_does_not_panic` ?

Also, I'm having trouble understanding how this test name relates to the PR description: what is the relevance of it being invalid? Is it that it is accepted? If so maybe `accepts_invalid_key_if_mac_and_iv_are_missing` ?

Reply:
```suggestion
    fn accepts_any_passphrase_if_mac_and_iv_are_missing() {
```